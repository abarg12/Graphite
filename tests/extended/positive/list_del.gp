list l = [1, 2.2, true, "four"];
string s = l.get(3);
printf(s);

l.del(2);
int i = l.len();
printf(i);

l.del(0);
i = l.len();
printf(i);

l.add(1, "hi");
string s2 = l.get(1);
printf(s2);
