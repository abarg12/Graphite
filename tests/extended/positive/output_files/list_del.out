four
2
1
hi
